src/app/component/ubs/components/ubs-confirm-page/ubs-confirm-page.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="wraper">
+    <div class="order">
+        <h2 class="title">Дякуємо за замовлення</h2>
+        <div class="description">
+            <p>
+                Ми вдячні Вам за бережливе ставлення до навколишнього середовища.
+                Якщо ви зацікавлені в інших послугах або корисній інформації, 
+                завітайте за нашими корисними посиланнями.
+            </p>
+        </div>
+        <div class="row infolinks">
+            <div class="col-md-6 list">
+                <ul>
+                    <h5>УБС сервіси</h5>
+                    <li><a>Зробити інше замовлення</a></li>
```

Comment:
move the text to appropriate translate file

Reply:
at the moment it is not advisable because the designers will completely redraw this page. It's just a mock for a redirect from the fondy. all content will be changed later